Source/Core/Core/PowerPC/Interpreter/Interpreter_Tables.cpp

```diff
@@ -88,10 +90,10 @@ static GekkoOPTemplate primarytable[] =
 	{61, Interpreter::psq_stu,      {"psq_stu", OPTYPE_STOREPS, FL_IN_FLOAT_S | FL_OUT_A | FL_IN_A | FL_USE_FPU | FL_LOADSTORE, 1, 0, 0, 0}},
 
 	//missing: 0, 1, 2, 5, 6, 9, 22, 30, 62, 58
-};
+}};
 
-static GekkoOPTemplate table4[] =
-{    //SUBOP10
+static std::array<GekkoOPTemplate, 13> table4 =
```

Comment:
Maybe not the right place to talk about it, but is duplicating the size information avoidable? Or does it compile-error if a row is deleted or added without updating the number?

(I ask because in C syntax, if you deleted a row, it would silently zero initialise the structure, and then iteration would probably replace ps_cmpu0 with a NULL pointer, and it would be horribly confusing to debug.)